include/zephyr/drivers/sensor.h

```diff
@@ -177,6 +177,9 @@ enum sensor_channel {
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
 
+	/** Ultraviolet Index (UVI) */
+	SENSOR_CHAN_UVI,
```

Comment:
Neat that this exists! I knew of UV intensity sensors for chromatography but not of UVI sensors.